addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_materials.py

```diff
@@ -60,6 +60,18 @@ def gather_material(blender_material, export_settings):
         pbr_metallic_roughness=__gather_pbr_metallic_roughness(blender_material, orm_texture, export_settings)
     )
 
+    print(material.pbr_metallic_roughness)
+    print("->", orm_texture)
+
+    # If emissive is set, but no orm_texture,
+    # "Emission" node is used. We need to set manually default values for 
+    # pbr_metallic_roughness.baseColor
+    if material.emissive_factor is not None and orm_texture is None:
```

Comment:
Is testing for orm_texture the correct answer here?  What if they have a Principled BSDF node without textures?

Reply:
Exact. I just pushed another (better) check